.github/workflows/nightly.yml

```diff
@@ -0,0 +1,98 @@
+name: Nightly build
+
+on:
+  schedule:
+    - cron: "0 3 * * *"
+  push:
+    branches:
+      - master
+      - main
+
+jobs:
+  build:
+    name: Build integration for
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        goos: [ linux ]
+        goarch: [ amd64, arm64, arm ]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '1.16'
+      - name: Build integration
+        env:
+          GOOS: ${{ matrix.goos }}
+          GOARCH: ${{ matrix.goarch }}
+        run: |
+          make compile
+      - name: Upload artifact for docker build step
+        uses: actions/upload-artifact@v2
+        with:
+          retention-days: 1
+          name: nri-kubernetes-${{ matrix.goos }}-${{ matrix.goarch }}
+          path: bin/nri-kubernetes-${{ matrix.goos }}-${{ matrix.goarch }}
+
+  docker:
+    name: Build and push images
+    needs: [ build ]
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_IMAGE_NAME: newrelic/infrastructure-k8s
+      DOCKER_IMAGE_TAG: latest-nightly
```

Comment:
Perhaps new repository should be created?

Reply:
@roobre

Reply:
personally I would go for a no on this

Reply:
Note, that this repository already has `latest` tag, similar to `infrastructure-bundle` and `infrastructure-bundle` images will be pushed to `infrastructure-bundle-nightly` repository, hence the question.

Reply:
`newrelic/infrastructure-k8s-nightly`

Reply:
We decided on call to push it to the same repository with `nightly` tag.